client/src/main/java/org/apache/rocketmq/client/impl/consumer/DefaultLitePullConsumerImpl.java

```diff
@@ -691,7 +691,7 @@ public void run() {
 
                 ProcessQueue processQueue = assignedMessageQueue.getProcessQueue(messageQueue);
 
-                if (processQueue == null && processQueue.isDropped()) {
+                if ((null == processQueue) || (processQueue != null && processQueue.isDropped())) {
```

Comment:
why not just write as `if (processQueue == null || processQueue.isDropped())` ? This is more concise

Reply:
ok,i change as you say